docs/eventing/broker-trigger.md

```diff
@@ -174,14 +174,12 @@ metadata:
   name: my-service
   namespace: default
 spec:
-  runLatest:
-    configuration:
-      revisionTemplate:
-        spec:
-          container:
-            # This corresponds to
-            # https://github.com/knative/eventing-contrib/blob/v0.2.1/cmd/message_dumper/dumper.go.
-            image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/message_dumper@sha256:ab5391755f11a5821e7263686564b3c3cd5348522f5b31509963afb269ddcd63
+  template:
```

Comment:
For each of these, don't you need to change the apiVerison to serving.knative.dev/v1beta1?

Reply:
Maybe not, since we ended up backing out the v1beta1 stuff?